packages/react-hooks/src/ConnectionProvider.tsx

```diff
@@ -15,21 +15,57 @@ import {
   addRecord,
 } from './recordUtils'
 
+export type useConnectionsOptions = {
```

Comment:
Types are generally PascalCase

```suggestion
export type UseConnectionsOptions = {
```